pkg/skaffold/trigger/fsnotify/hidden_posix.go

```diff
@@ -0,0 +1,38 @@
+// +build !windows
+
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fsnotify
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// hidden checks if the file is hidden or in hidden dir.
+// notify.EventInfo.Path always returns absolute paths.
```

Comment:
maybe add a comment that this doesn't accept relative filepaths like `./foo.txt` or `../foo.txt`

Reply:
🤦🏽‍♂️ I didn't see the comment above.